app/Models/User.php

```diff
@@ -32,6 +32,7 @@
  * @property string|null                                           $email
  * @property bool                                                  $may_upload
  * @property bool                                                  $is_locked
+ * @property string|null                                           $token
```

Comment:
The `has_token` property is missing. Moreover `'token'` still needs to be added to the array `$hidden` to prevent sending it to the frontend.

Reply:
Fixed this in https://github.com/LycheeOrg/Lychee/pull/1368/commits/871eb59816a065adc81200fec7ce1883de73d1c7 myself. IMHO this can be resolved.